src/sugar.jl

```diff
@@ -234,10 +236,3 @@ function print_in_atlens(io, l)
     end
     print(io, ')')
 end
-
-function show_generic(io::IO, args...)
```

Comment:
A nice side-effect was that we can get rid of `show_generic` now.

Reply:
Yes that is great!